common/sgx/qeidentity.h

```diff
@@ -12,13 +12,34 @@
 
 OE_EXTERNC_BEGIN
 
-oe_result_t oe_enforce_qe_identity(sgx_report_body_t* qe_report_body);
+/**
+ * This is needed to be backwards compatible
+ * with the older quote provider.
+ *
```

Comment:
-> "with platform quote providers that do not support qe identity"

Reply:
Thanks, it's clearer now.